src/components/atoms/Headline/Headline.tsx

```diff
@@ -0,0 +1,19 @@
+import { css } from "@emotion/css"
+
+type HeadlineProps = {
+  headlineText: string
+}
+
+export const Headline: React.FC<HeadlineProps> = ({headlineText}) => {
+    return (
+        <h2 className={styles.container}> {headlineText} </h2>
```

Comment:
このスペースおそらくレンダリングされちゃう気がする！
```suggestion
        <h2 className={styles.container}>{headlineText}</h2>
```